src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs

```diff
@@ -2,9 +2,10 @@
 #if DNX451
 using System.Runtime.Remoting;
 using System.Runtime.Remoting.Messaging;
-#endif
+#else
 using System.Threading;
```

Comment:
System.Threading seems to be always included before this change.  Not it's not included for DNX451.  Why this change?